index.bs

```diff
@@ -1406,7 +1406,7 @@ Instances of <code>WritableStream</code> are created with the internal slots des
 <ol>
   <li> Assert: <var>stream</var>@\[[state]] is <code>"closing"</code>.
   <li> Let <var>sinkClosePromise</var> be PromiseInvokeOrNoop(<var>stream</var>@\[[underlyingSink]],
-    <code>"close"</code>).
+    <code>"close"</code>, «‍»).
```

Comment:
No need for empty, at least, not in ES Invoke... I'll make sure PromiseInvokeOrNoop also doesn't need empty lists.